src/Analysis/Ast/Impl/Modules/CompiledPythonModule.cs

```diff
@@ -60,51 +52,18 @@ public override string Documentation
                     return string.Empty;
                 }
 
-                code = ScrapeModule();
+                var args = GetScrapeArguments(Interpreter);
+                if (args == null) {
+                    return string.Empty;
+                }
+
+                code = StubGenerator.Scrape(Interpreter, Log, this, args, CancellationToken.None);
```

Comment:
It may better to put scraper in `Python.Core` perhaps so analysis won't depend on generator and rather the other way around. This way analysis gets scraper functionality from core and so does stub generator.